how-to-guides/re-run-failed-training/notebooks/re_run_failed_training.ipynb

```diff
@@ -0,0 +1,489 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "THIzFFk81nme"
+   },
+   "source": [
+    "# How to re-run failed training\n",
+    "\n",
+    "## Before you start\n",
```

Comment:
"Before you start" has only libs installation, so I would go with a bit simpler structure, that is:

```
# How to re-run failed training
**Install dependencies**
<code>

## Step 1 Get Run ID
...
...
```